insights/specs/default.py

```diff
@@ -169,6 +169,7 @@ def docker_container_ids(broker):
     docker_sysconfig = simple_file("/etc/sysconfig/docker")
     dumpdev = simple_command("/bin/awk '/ext[234]/ { print $1; }' /proc/mounts")
     dumpe2fs_h = foreach_execute(dumpdev, "/sbin/dumpe2fs -h %s")
+    engine_config_all = simple_command("/usr/bin/engine-config --all")
```

Comment:
In 3.x, for simple_command, should add its `simple_file` to the `specs/insights_archives.py`. You can use the 'bin/mangle' command to generate it.

~~~
mangle "/usr/bin/engine-config --all"
~~~

Reply:
@xiangce I don't find a guide to create rule which has example as this in http://insights-core.readthedocs.io/en/latest/. Where is this documented?

Reply:
Besides, what is the package which provides `mangle` command? 

I see "no match found" on RHVH and below output on Fedora,
```
$ sudo dnf whatprovides */mangle
Last metadata expiration check: 0:07:36 ago on Wed 14 Mar 2018 03:18:16 PM IST.
shorewall-5.1.10.2-1.fc27.noarch : An iptables front end for firewall configuration
Repo        : updates
Matched from:
Filename    : /etc/shorewall/mangle
Matched from:
Filename    : /usr/share/shorewall/configfiles/mangle

shorewall6-5.1.10.2-1.fc27.noarch : Files for the IPV6 Shorewall Firewall
Repo        : updates
Matched from:
Filename    : /etc/shorewall6/mangle
Matched from:
Filename    : /usr/share/shorewall6/configfiles/mangle

shorewall-5.0.14.1-5.fc27.noarch : An iptables front end for firewall configuration
Repo        : fedora
Matched from:
Filename    : /etc/shorewall/mangle
Matched from:
Filename    : /usr/share/shorewall/configfiles/mangle

shorewall6-5.0.14.1-5.fc27.noarch : Files for the IPV6 Shorewall Firewall
Repo        : fedora
Matched from:
Filename    : /etc/shorewall6/mangle
Matched from:
Filename    : /usr/share/shorewall6/configfiles/mangle
```